File: Assignment_2/Ass2_1.c
#include<stdio.h>

void Display(int iNo1)
{
    int iCnt = 0;

    while(iCnt < iNo1)
    {
        printf("*");
        iCnt++;
    }

}
int main()
{
    int iValue = 0;

    printf("Enter a Number : \n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Java_Assignments/Assignment_2/Ass2_1.java
import java.util.*;

class Ass2_1
{
	public void Display(int iNo)
	{
		int iCnt = 0;
		
		while(iCnt < iNo
		{
			System.out.print(" * ");
			iCnt++;
		}
	}
	
	public static void main(String A[])
	{
		int iValue = 0;
		
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter value : ");
		iValue = sobj.nextInt();
		
		Ass2_1 aobj = new Ass2_1();
		
		aobj.Display(iValue);
	}	
}